include/staff/templates/2fas.tmpl.php

```diff
@@ -18,19 +18,25 @@
 <table class="table">
     <tbody>
       <?php
-      $config = $staff->getConfig();
       foreach (Staff2FABackend::allRegistered() ?: array() as $bk) {
-          $isVerified = (isset($config[$bk->getId()]) &&
-                  $config[$bk->getId()]['verified']);
+          $configuration = $staff->get2FAConfig($bk->getId());
+          if ($isVerified = $configuration['verified']) {
           ?>
+          <script type="text/javascript">
+              isVerified = '<?php echo $isVerified;?>';
+              id = '<?php echo $id;?>';
+              if (id != 0)
+                  enable2fa(isVerified, id);
```

Comment:
js script in a foreach loop worries me here. I understand the goal is to enable just configured 2fa - but  I think it can be done a little more cleanly.